From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 Dec 2021 16:49:05 +0100
Subject: [PATCH] {MAYBE UPS} net/netfilter/nft_set_pipapo_avx2: remove lt
 variable

clang on x86-64 reports that it is unused but set.
---
 net/netfilter/nft_set_pipapo_avx2.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/netfilter/nft_set_pipapo_avx2.c b/net/netfilter/nft_set_pipapo_avx2.c
index e517663e0cd1..f84685d43b23 100644
--- a/net/netfilter/nft_set_pipapo_avx2.c
+++ b/net/netfilter/nft_set_pipapo_avx2.c
@@ -1048,10 +1048,10 @@ static int nft_pipapo_avx2_lookup_slow(unsigned long *map, unsigned long *fill,
 					struct nft_pipapo_field *f, int offset,
 					const u8 *pkt, bool first, bool last)
 {
-	unsigned long *lt = f->lt, bsize = f->bsize;
+	unsigned long /* *lt = f->lt,*/ bsize = f->bsize;
 	int i, ret = -1, b;
 
-	lt += offset * NFT_PIPAPO_LONGS_PER_M256;
+	/*lt += offset * NFT_PIPAPO_LONGS_PER_M256;*/
 
 	if (first)
 		memset(map, 0xff, bsize * sizeof(*map));
-- 
